documentation/docs/getting_started/python.mdx

```diff
@@ -0,0 +1,117 @@
+---
+description: 'Get started with the official IOTA Wallet Library Python binding.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- Python
+- install
+- pip
+- unpack
+---
+import SafePasswordStorage from '../_admonitions/_warning_safe_password_storage.md'
+
+# Getting Started with Python
+
+<SafePasswordStorage />
+
+## Requirements
+
+* [Python 3.x](https://www.python.org)
+* [pip](https://pypi.org/project/pip) >= 19.1
+* [setuptools-rust](https://pypi.org/project/setuptools-rust/) >= 0.10.2
+* [Rust and Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html) to compile the binding.
```

Comment:
Same here, this also need the dependencies listed in the Rust getting started

Reply:
Would something like this work?

```suggestion
* Make sure you have fulfilled the [Rust Getting Started](rust.mdx) requirements to compile the binding.
```